src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -722,7 +722,7 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
                         : ''
                 }`}
             >
-                {(this.state.isRollbackTrigger || this.state.isSelectImageTrigger) && !this.state.showConfigDiffView && (
+                {(this.state.isRollbackTrigger || this.state.isSelectImageTrigger) && !this.state.showConfigDiffView && !(cdModalHeader.some(heading=> this.renderCDModalHeader()===heading)) &&(
```

Comment:
Can we add this check based on stageType variable instead of creating new cdModalHeader array